<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BracketOrderGui</class>
 <widget class="QDialog" name="BracketOrderGui">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>700</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>1000</width>
    <height>700</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1000</width>
    <height>700</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Bracket Order</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_4">
       <property name="maximumSize">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
       <property name="title">
        <string/>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_13">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_12">
          <item>
           <widget class="QComboBox" name="sourceComboBox"/>
          </item>
          <item>
           <widget class="QLineEdit" name="partCodeLineEdit"/>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonClose">
       <property name="minimumSize">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
       <property name="text">
        <string>-</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <widget class="QGroupBox" name="groupBoxShowHide">
     <property name="title">
      <string/>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_20">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_19">
        <item>
         <widget class="QGroupBox" name="groupBox_2">
          <property name="title">
           <string>Symbol Leg 1</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_15">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_3">
             <item>
              <layout class="QVBoxLayout" name="verticalLayout">
               <item>
                <widget class="QLabel" name="label">
                 <property name="text">
                  <string>Mode</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QComboBox" name="orderComboBox"/>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QVBoxLayout" name="verticalLayout_2">
               <item>
                <widget class="QLabel" name="label_2">
                 <property name="text">
                  <string>Market</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QComboBox" name="marketComboBox"/>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QVBoxLayout" name="verticalLayout_3">
               <item>
                <widget class="QLabel" name="label_3">
                 <property name="text">
                  <string>Instrument</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QComboBox" name="instrumentComboBox"/>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QVBoxLayout" name="symbolVerticalLayout">
               <item>
                <widget class="QLabel" name="label_4">
                 <property name="text">
                  <string>Symbol</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QVBoxLayout" name="verticalLayout_5">
               <item>
                <widget class="QLabel" name="label_5">
                 <property name="text">
                  <string>Series</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QComboBox" name="seriesComboBox"/>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QVBoxLayout" name="verticalLayout_6">
               <item>
                <widget class="QLabel" name="label_6">
                 <property name="text">
                  <string>Expiry</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QComboBox" name="expiryComboBox"/>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QVBoxLayout" name="verticalLayout_7">
               <item>
                <widget class="QLabel" name="label_7">
                 <property name="text">
                  <string>Opt Mode</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QComboBox" name="optModeComboBox"/>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QVBoxLayout" name="verticalLayout_8">
               <item>
                <widget class="QLabel" name="label_8">
                 <property name="text">
                  <string>Strike</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QComboBox" name="strikeComboBox"/>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QVBoxLayout" name="verticalLayout_9">
               <item>
                <widget class="QLabel" name="label_9">
                 <property name="text">
                  <string>Lot Size</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="lotSizeLineEdit">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="maximumSize">
                  <size>
                   <width>85</width>
                   <height>16777215</height>
                  </size>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QVBoxLayout" name="verticalLayout_10">
               <item>
                <widget class="QLabel" name="label_10">
                 <property name="text">
                  <string>Account</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QGroupBox" name="groupBox_3">
                 <property name="title">
                  <string/>
                 </property>
                 <layout class="QVBoxLayout" name="verticalLayout_14">
                  <item>
                   <layout class="QHBoxLayout" name="horizontalLayout_2">
                    <item>
                     <widget class="QRadioButton" name="proRadioButton">
                      <property name="text">
                       <string>Pro</string>
                      </property>
                     </widget>
                    </item>
                    <item>
                     <widget class="QRadioButton" name="clientRadioButton">
                      <property name="text">
                       <string>Client</string>
                      </property>
                     </widget>
                    </item>
                   </layout>
                  </item>
                 </layout>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QVBoxLayout" name="verticalLayout_11">
               <item>
                <widget class="QLabel" name="label_11">
                 <property name="text">
                  <string/>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QComboBox" name="clientCodeComboBox"/>
               </item>
              </layout>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox_5">
          <property name="title">
           <string>Strategy Params</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_16">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_19">
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_29">
               <item>
                <widget class="QLineEdit" name="modeLeg1LineEdit">
                 <property name="maximumSize">
                  <size>
                   <width>80</width>
                   <height>16777215</height>
                  </size>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="label_36">
                 <property name="maximumSize">
                  <size>
                   <width>100</width>
                   <height>16777215</height>
                  </size>
                 </property>
                 <property name="text">
                  <string>Order Type Leg1</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QComboBox" name="orderLeg1ComboBox"/>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_28">
               <item>
                <widget class="QLabel" name="label_35">
                 <property name="text">
                  <string>Fill Type Leg1</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QComboBox" name="fillLeg1ComboBox"/>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_27">
               <item>
                <widget class="QLabel" name="label_34">
                 <property name="text">
                  <string>Quantity Leg1</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QSpinBox" name="quantityLeg1SpinBox">
                 <property name="maximumSize">
                  <size>
                   <width>70</width>
                   <height>16777215</height>
                  </size>
                 </property>
                 <property name="maximum">
                  <number>1000000000</number>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_26">
               <item>
                <widget class="QLabel" name="label_33">
                 <property name="text">
                  <string>Limit Price Leg1</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QDoubleSpinBox" name="limitLeg1DoubleSpinBox">
                 <property name="decimals">
                  <number>2</number>
                 </property>
                 <property name="maximum">
                  <double>10000000.000000000000000</double>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_25">
               <item>
                <widget class="QLabel" name="label_32">
                 <property name="text">
                  <string>Trigger Price Leg1</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QDoubleSpinBox" name="triggerLeg1DoubleSpinBox">
                 <property name="maximum">
                  <double>10000000.000000000000000</double>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_4">
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_9">
               <item>
                <widget class="QLineEdit" name="modeLeg2LineEdit">
                 <property name="maximumSize">
                  <size>
                   <width>80</width>
                   <height>16777215</height>
                  </size>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="label_17">
                 <property name="maximumSize">
                  <size>
                   <width>100</width>
                   <height>16777215</height>
                  </size>
                 </property>
                 <property name="text">
                  <string>Order Type Leg2</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QComboBox" name="orderLeg2ComboBox">
                 <property name="maximumSize">
                  <size>
                   <width>16777215</width>
                   <height>16777215</height>
                  </size>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_10">
               <item>
                <widget class="QLabel" name="label_18">
                 <property name="text">
                  <string>Fill Type Leg2</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QComboBox" name="fillLeg2ComboBox"/>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_11">
               <item>
                <widget class="QLabel" name="label_19">
                 <property name="text">
                  <string>Quantity Leg2</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QComboBox" name="quantityLeg2ComboBox">
                 <property name="maximumSize">
                  <size>
                   <width>16777215</width>
                   <height>16777215</height>
                  </size>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_12">
               <item>
                <widget class="QLabel" name="label_20">
                 <property name="text">
                  <string>Limit Price Leg2</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QDoubleSpinBox" name="limitLeg2DoubleSpinBox">
                 <property name="maximum">
                  <double>10000000.000000000000000</double>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_13">
               <item>
                <widget class="QLabel" name="label_21">
                 <property name="text">
                  <string>Trigger Price Leg2</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QDoubleSpinBox" name="triggerLeg2DoubleSpinBox">
                 <property name="maximum">
                  <double>10000000.000000000000000</double>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_5">
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_14">
               <item>
                <widget class="QLineEdit" name="modeLeg3LineEdit">
                 <property name="maximumSize">
                  <size>
                   <width>80</width>
                   <height>16777215</height>
                  </size>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="label_22">
                 <property name="maximumSize">
                  <size>
                   <width>100</width>
                   <height>16777215</height>
                  </size>
                 </property>
                 <property name="text">
                  <string>Order Type Leg3</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QComboBox" name="orderLeg3ComboBox"/>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_15">
               <item>
                <widget class="QLabel" name="label_23">
                 <property name="text">
                  <string>Fill Type Leg3</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QComboBox" name="fillLeg3ComboBox"/>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_16">
               <item>
                <widget class="QLabel" name="label_24">
                 <property name="text">
                  <string>Quantity Leg3</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QComboBox" name="quantityLeg3ComboBox"/>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_17">
               <item>
                <widget class="QLabel" name="label_25">
                 <property name="text">
                  <string>Limit Price Leg3</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QDoubleSpinBox" name="limitLeg3DoubleSpinBox">
                 <property name="maximum">
                  <double>10000000.000000000000000</double>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_18">
               <item>
                <widget class="QLabel" name="label_26">
                 <property name="text">
                  <string>Trigger Price Leg3</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QDoubleSpinBox" name="triggerLeg3DoubleSpinBox">
                 <property name="maximum">
                  <double>10000000.000000000000000</double>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0">
    <layout class="QVBoxLayout" name="buttonPanel"/>
   </item>
   <item row="3" column="0">
    <layout class="QVBoxLayout" name="tablePanel"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
